sdk/go/kubernetes/helm/v3/chartPulumiTypes.go

```diff
@@ -0,0 +1,210 @@
+// Copyright 2016-2021, Pulumi Corporation.
```

Comment:
This is a rename to emit release in the same package as helm/v3. I waffled back and forth but this scheme seems to the best fit for future major versions for Helm.